chore(docs): add tiiny host tutorial (#22284)
gatsbyjs/gatsby: docs/docs/deploying-to-tiiny-host.md

---
title: Deploying to tiiny.host
---

In this guide, you will learn how to deploy your Gatsby site to [tiiny.host](https://tiiny.host).

[tiiny.host](https://tiiny.host/) is a tool for quickly hosting static websites.

> The tool is perfect for prototyping and sharing your gatsby site online in a few clicks.

Reviewer: ```suggestion
The tool is perfect for prototyping and sharing your Gatsby site online in a few clicks.
```

This guide will show you how to deploy in two steps:

## Step 1: Prepare your build

Build your site by running this command in your project's root directory:

```shell
gatsby build
```

This will generate a publishable version of your site in the `./public` folder.

## Step 2: Deploy

> Zip your `./public` folder and head over to [https://tiiny.host](https://tiiny.host)

Reviewer: ```suggestion
Zip your `./public` folder and head over to [https://tiiny.host](https://tiiny.host).
```

> Here, enter a subdomain, upload your zip and click `Launch`.

Reviewer: ```suggestion
Here, enter a subdomain, upload your zip file and click `Launch`.
```

> And that's it, You're done! Theres no need to register and your site will be live for 7 days or can be extended on the Pro plan.

Reviewer: ```suggestion
And that's it, you're done! There is no need to register and your site will be live for 7 days or can be extended on the Pro plan.
```

Author: Good spot! I've update the docs.
